examples/gallery/plot/multi_parameters.py

```diff
@@ -0,0 +1,62 @@
+"""
+Multi-parameter symbols
+----------------
+
+The :meth:`pygmt.Figure.plot` method can plot individual multi-parameter symbols by passing 
+the corresponding shortcuts listed below to the ``style`` option. Additionally, we must define 
+the required parameters in a numpy array (or use an appropriately formatted input file) and pass it 
```

Comment:
This is not 100% correct. A 2D list should also work.

Reply:
> 
> 
> This is not 100% correct. A 2D list should also work.

It needs to be always a 2D array right?

Reply:
> It needs to be always a 2D array, right?

I think so.

Reply:
> 
> 
> > It needs to be always a 2D array, right?
> 
> I think so.

ok fine